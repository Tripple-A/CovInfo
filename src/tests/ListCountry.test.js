import React from 'react';
import ReactDom from 'react-dom';
import { render } from '@testing-library/react';
import ListCountry from '../components/showCountries';

it('renders without crashing', () => {
  const div = document.createElement('div'); 
});
