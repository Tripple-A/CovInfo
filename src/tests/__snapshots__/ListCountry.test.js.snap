// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <a
    href="/covinfo/France"
    onClick={[Function]}
  >
    <div
      className="country"
    >
      <div
        className="flag"
      >
        <img
          alt="country-flag"
          src="https://www.countryflags.io/FR/flat/64.png"
        />
      </div>
      <div
        className="country-name"
      >
        <h3
          data-testid="name"
        >
          France
        </h3>
      </div>
    </div>
  </a>
</div>
`;
